lib/davinci_us_drug_formulary_test_kit/custom_groups/v2.0.1/visual_inspection_and_attestation/drug_display_name.rb

```diff
@@ -16,20 +16,41 @@ class FormularyDrugNameSearchDisplayTest < Inferno::Test
 
       verifies_requirements 'hl7.fhir.us.davinci-drug-formulary_2.0.1@19'
 
-      run do
-        identifier = SecureRandom.hex(32)
+      input :usdf_v201_drug_display_name_options,
+            title: 'Supports drug name search using general drug form group codes and names',
+            description: %(
+              I attest that the Health IT Module supports searching for drugs using general drug form
```

Comment:
make consistent with the others?

```suggestion
              The developer of the Health IT Module attests that the system supports searching for drugs using general drug form
```